src/main/java/com/imsweb/mph/mpgroups/Mp2004SolidMalignantGroup.java

```diff
@@ -23,9 +24,8 @@ public Mp2004SolidMalignantGroup() {
         MphRule rule = new MphRule(MphConstants.MP_2004_SOLID_MALIGNANT, "M1") {
             @Override
             public TempRuleResult apply(MphInput i1, MphInput i2) {
-                TempRuleResult result = new TempRuleResult();
                 //TODO
```

Comment:
Are you planning on addressing those TODO in another Pull Request? I see some others in this class...

Reply:
Those are rules which can't be automated. For solid tumors M1 and M2 usually are just a pre-mph rules which we can't automate. I just left the TODOs to see if it is possible to automate them at some point.

Reply:
I think it would be better to submit an issue and explain what you would like to do and assign it to yourself. TODOs are so anonymous and unclear. Could you remove them and submit an issue instead?

Reply:
This is fixed!